tldw_Server_API/app/api/v1/endpoints/writing_manuscripts.py

```
@@ -0,0 +1,816 @@
+        _handle_db_errors(exc, "manuscript structure")
+
+
+@router.post(
+    "/projects/{project_id}/reorder",
+    status_code=status.HTTP_204_NO_CONTENT,
+    response_class=Response,
+    summary="Batch reorder parts, chapters, or scenes",
+    tags=["manuscripts"],
+)
+async def reorder_entities(
+    project_id: str,
```

Ruff (0.15.9)

[warning] Unused function argument: `project_id`

(ARG001)